tests/ServerTest.php

```diff
@@ -267,6 +267,121 @@ public function testRequestNonConnectWithAuthorityRequestTargetWillReject()
         $this->connection->emit('data', array($data));
     }
 
+    public function testRequestAbsoluteEvent()
+    {
+        $requestAssertion = null;
+
+        $server = new Server($this->socket, function (RequestInterface $request) use (&$requestAssertion) {
+            $requestAssertion = $request;
+            return new Response();
+        });
+
+        $this->socket->emit('connection', array($this->connection));
+
+        $data = "GET http://example.com/test HTTP/1.1\r\nHost: example.com\r\n\r\n";
+        $this->connection->emit('data', array($data));
+
+        $this->assertInstanceOf('RingCentral\Psr7\Request', $requestAssertion);
+        $this->assertSame('GET', $requestAssertion->getMethod());
+        $this->assertSame('http://example.com/test', $requestAssertion->getRequestTarget());
+        $this->assertEquals('http://example.com/test', $requestAssertion->getUri());
+        $this->assertSame('/test', $requestAssertion->getUri()->getPath());
+        //$this->assertSame(array(), $requestAssertion->getQueryParams());
```

Comment:
Any reason this is commented out? To me commented out code shouldn't be committed tbh